lib/Chat/Notifier.php

```diff
@@ -314,8 +314,16 @@ public function removePendingNotificationsForRoom(Room $chat, bool $chatOnly = f
 		// @todo this should be in the Notifications\Hooks
 		$notification->setApp('spreed');
 
-		$notification->setObject('chat', $chat->getToken());
-		$this->notificationManager->markProcessed($notification);
+		$objectTypes = [
```

Comment:
This function is called when the chat history is cleared, not sure we want to remove recording notifications and information at that case?

Reply:
If we consider that the notification is part of chat history, yes.
This method also is called when the room is deleted and in this case, is necessary to delete the notification because all data related of the room is deleted.

Reply:
Well there is a `$chatOnly` parameter and that is the difference, but yeah maybe it's fine.

Reply:
Is this new notification at context of `$chatOnly`, right?